tests/org.jboss.tools.maven.ui.bot.test/src/org/jboss/tools/maven/ui/bot/test/AbstractMavenSWTBotTest.java

```diff
@@ -256,6 +258,18 @@ public void createWebProject(String name,String runtime, boolean webxml){
 		} else {
 			dfp.setTargetRuntime(runtime);
 		}
+		
+		// update default configuration for Java 8
+		if (getJavaVersion().equals("1.8")) {
+			new PushButton("Modify...").click();
+			Tree tree = new DefaultTree();
+			tree.getItem("Java").select();
+			tree.getContextMenu().getItem("Change Version...").select();
+			new DefaultCombo(0).setSelection("1.8");
+			new PushButton("OK").click();
+			new PushButton("OK").click();
```

Comment:
Its not a bug: you can use new OkButtont()

Reply:
Replaced.